src/helpers/sorts.ts

```diff
@@ -103,6 +103,16 @@ export function getFirstMeasureSort(afm: AFM.IAfm): AFM.SortItem[] {
     return [];
 }
 
+export function getDefaultAttributeGeoPushpinSort(afm: AFM.IAfm): AFM.SortItem[] {
+    const attributes: AFM.IAttribute[] = get(afm, "attributes");
+
+    if (attributes.length === 2 || attributes.length === 3) {
```

Comment:
how about `if (attributes.length > 1)`? 
Please add some comment here about this sort

Reply:
fixed